controller/controller.go

```diff
@@ -220,14 +220,26 @@ func (c *Controller) RunOnce(ctx context.Context) error {
 		ManagedRecords:     c.ManagedRecordTypes,
 	}
 
-	plan = plan.Calculate()
+	p = p.Calculate()
 
-	if plan.Changes.HasChanges() {
-		err = c.Registry.ApplyChanges(ctx, plan.Changes)
+	if p.Changes.HasChanges() {
+		err = c.Registry.ApplyChanges(ctx, p.Changes)
 		if err != nil {
-			registryErrorsTotal.Inc()
-			deprecatedRegistryErrors.Inc()
-			return err
+			changes := append(p.Changes.Create, p.Changes.Delete...)
+			changes = append(changes, p.Changes.UpdateNew...)
+			changes = append(changes, p.Changes.UpdateOld...)
+
+			for _, change := range changes {
+				cg := &plan.Changes{
+					Create: []*endpoint.Endpoint{change},
```

Comment:
We cannot merge the deletions and updates into the creations. They are all of the same type but the respective slice defines what should happen with them, e.g. here we would now try to _create_ records that should be _deleted_.